src/index.ts

```diff
@@ -15,17 +26,44 @@ const io = new SocketIOServer(httpServer, {
   },
 });
 
-app.get('/', (req: Request, res: Response) => {
-  res.send('Hello World!');
+
+app.post('/generate-requirements', upload.single('textFile'), async (req: Request, res: Response) => {
+  if (!req.file) {
+    return res.status(400).send('There was no file uploaded.');
+  }
+
+  const fileContent: Buffer = req.file.buffer;
+  const projectBrief: string = fileContent.toString();
```

Comment:
this should be called something else since `projectBrief` is used specifically for something else in our application context